src/helpers/spaces.ts

```diff
@@ -1,24 +1,22 @@
-import { getActiveProposals, getFollowers } from './adapters/mysql';
+import { getProposals, getFollowers } from './adapters/mysql';
 import db from './mysql';
 
 export let spaces = {};
-export const spacesActiveProposals = {};
+export const spaceProposals = {};
 export const spaceFollowers = {};
 
 export const spaceIdsFailed: string[] = [];
 
 setInterval(() => {
-  getActiveProposals().then((result: any) =>
-    result.forEach(count => {
-      if (spaces[count.space]) {
-        spaces[count.space]._activeProposals = count.count;
-        spacesActiveProposals[count.space] = count.count;
+  getProposals().then((result: any) =>
+    result.forEach(proposals => {
+      if (spaces[proposals.space]) {
+        spaces[proposals.space]._activeProposals = proposals.active;
```

Comment:
Can you remove `_activeProposals` from the `spaces` var, this is not used anymore